include/dts-subscription.sh

```diff
@@ -90,7 +90,7 @@ get_dpp_creds() {
 login_to_dpp_server(){
   # Check if the user is already logged in, log in if not:
   if [ -z "$(mc alias list | grep ${CLOUDSEND_DOWNLOAD_URL})" ]; then
-    mc alias set $DPP_SERVER_USER_ALIAS $DPP_SERVER_ADDRESS $CLOUDSEND_DOWNLOAD_URL $CLOUDSEND_PASSWORD || return 1
+    mc alias set $DPP_SERVER_USER_ALIAS $DPP_SERVER_ADDRESS $CLOUDSEND_DOWNLOAD_URL $CLOUDSEND_PASSWORD 2> /dev/null || return 1
```

Comment:
Now the actual error is stored in `ERR_LOG_FILE` and the warning message is printed to the script. For some reason it's being printed twice though. 3289fb68d3f51ca50a4a1b68262bafee1ac1b639

Reply:
This is because it is being used twice, you can check it with `grep`. Unfortunately, I see no way we can handle this now, so lets stay with printing it twice.